app/components/customDateTimePicker/dateTimePickerWrapper.tsx

```diff
@@ -0,0 +1,111 @@
+import { DateTimePicker as ReactDateTimePicker } from 'react-datetime-picker'
+import { useColorMode } from '@chakra-ui/react'
+import { MdCalendarToday, MdClose } from 'react-icons/md'
+import 'react-clock/dist/Clock.css'
+import 'react-datetime-picker/dist/DateTimePicker.css'
+import 'react-calendar/dist/Calendar.css'
+
+type ValuePiece = Date | null
+type Value = ValuePiece | [ValuePiece, ValuePiece]
+
+export const DateTimePickerWrapper = ({
+  onChange,
+  value
+}: {
+  onChange: (value: Value) => void
+  value: Value
+}) => {
+  const { colorMode } = useColorMode()
+
+  return (
+    <>
+      <style>{`
```

Comment:
For darkmode, the background color is too white when you hover? 
<img width="572" alt="Screenshot 2024-01-04 at 12 51 40 PM" src="https://github.com/tabi-memo/frontend/assets/66394413/964bca42-15dd-4c79-be37-1f7979394135">

And if you can, I think it's good to place under the input so that the user can see the input at the same time. like this!
<img width="553" alt="Screenshot 2024-01-04 at 12 52 48 PM" src="https://github.com/tabi-memo/frontend/assets/66394413/666ac6b0-97e5-4b7c-9815-5c5f01f6e66b">

Reply:
https://github.com/tabi-memo/frontend/assets/70562492/93b1d70e-0192-4664-ab0d-1a10d86f03be

It's fixed :)